src/main/groovy/io/gatling/gradle/GatlingEnterpriseStartTask.groovy

```diff
@@ -35,6 +37,8 @@ class GatlingEnterpriseStartTask extends DefaultTask {
                          |Simulation ${simulationStartResult.simulation.name} successfully started.
                          |Once running, reports will be available at: ${gatling.enterprise.url.toExternalForm() + simulationStartResult.runSummary.reportsPath}
                          |""".stripMargin())
+        } catch (UserQuitException e) {
+            throw new BuildCancelledException(e.getMessage(), e)
```

Comment:
I think it makes sense to throw a `BuildCancelledException` in this case, but note that in practice it's displayed just like any other exception to the user.

We could also simply log the message and finish the task, but that would mean exiting with a return code `0`, which is weird if the task was interrupted.